src/components/Courses/components/CourseCard/Author.types.ts

```diff
@@ -0,0 +1,8 @@
+export interface Author {
+	id: string;
+	name: string;
+}
+
+export function buildAuthor(id: string, name: string): Author {
```

Comment:
seems that this instance is not fully a type)

Reply:
fixed